src/web_app/css/main.css

```diff
@@ -104,6 +104,22 @@ footer > div.active {
   opacity: .8;
 }
 
+div#privacy {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  /* Makes sure to stay behind room links so they can be clicked */
+  z-index: -1
+  margin-right: 1em;
+  margin-bottom: 0.5em;
+  margin-top: calc(100vh - 34em);
+}
+
+div#privacy a {
```

Comment:
I'd leave these links blue, like other links, and not use margin (just add a space a pipe between them).